### ISSUE TITLE
Accessibility issue on using 'i' tags for icons

### ISSUE THREAD

**Issue by viswanathamsantosh** [mozilla/addons-frontend#3376]
### Describe the problem and steps to reproduce it:
Just found that the project has been using `i` tags for using icons. But that doesn't comply with accessibility standards. `i` tags should be used only to make a text italic. May be we could replace `i`  with `span` tags all over the project. Wont be needing any difference in styling but also improves the accessibility of AMO. what do you think?

**Comment by tofumatt** [comment]
I’m fine with this. I forget why we choose i tags but they don’t really make sense. A span should do fine... the only issue might be icons that render inside spans which I think html doesn’t like. 😐

Matthew Riley MacPherson (sent from mobile)
